tmt/steps/provision/testcloud.py

```diff
@@ -1,4 +1,5 @@
 # coding: utf-8
+# mypy: disable-error-code="name-defined"
```

Comment:
I'm not sure we should take this road, disabling a rule like this globally can easily prevent mypy from spotting actual errors. Is this about the `global *` objects imported from `testcloud` package, the amount of possible `type: ignore[...]` needed to silence them all?

Reply:
Resolved, it certainly doesn't look nice though...

Reply:
I suppose these stem out of not being declared on the module level, unlike `libvirt` and `testcloud`. mypy is unable to realize our code would call `import_testcloud()` that would set these "global" names. Would you mind declaring them on the module level, I suppose `DomainConfiguration: Any`, without any value, should introduce the name well enough for mypy to not require `name-defined` anymore?

Reply:
Yep, that did the trick! Thanks!

(Took me a while before I'd realized I need to import the Any type... :) )